eng/pipelines/live-test-cleanup.yml

```diff
@@ -1,6 +1,14 @@
+schedules:
+# At minute 15 past every hour
+- cron: "15 */1 * * *"
+  displayName: Hourly
```

Comment:
Is hourly needed? Should we do it daily? Perhaps we can look at some recent runs to see how much clean-up these actually do and then decide if that hourly is needed, but if it is then I think we likely have other issues we need to address.

Reply:
Just peeked through the last ~month of runs. Once a day is plenty.

Reply:
Display name still says Hourly.